content/reference/endpoints/rentals.md

```diff
@@ -19,15 +19,18 @@ id                              | Integer  | Read       | Rental's id.
 absolute_min_price              | [Decimal](/reference/enums#formats) | Read       | Rental's minimum price based on `base_rate_kind`.
 base_rate                       | [Decimal](/reference/enums#formats) | Read       | Rental's base rate, used by the rates table to calculate the seasonability and discounted prices. We recommend using the highest price as base rate.
 base_rate_kind                  | String   | Read       | Rental's rate type attached to the base rate. List of types described in [enums section](/reference/enums#rates-types).
+balance_due                     | Integer  | Read       | Number of day since the booking confirmation to collect the full booking price.
 bathrooms_count                 | Integer  | Read       | Rental's bathrooms count.
 bedrooms_count                  | Integer  | Read       | Rental's bedrooms count.
 bookable_online                 | Boolean  | Read       | Rental's online booking status.
 checkin_time                    | Integer  | Read       | Rental's checkin time.
+checkin_end_time                | Integer  | Read       | Rental's checkin end time.
 checkout_time                   | Integer  | Read       | Rental's checkout time.
 city                            | String   | Read       | Rental's city.
 contact_name                    | String   | Read       | Rental's contact name.
 country_code                    | String   | Read       | Rental's country code, list of codes is available at [wikipedia](http://en.wikipedia.org/wiki/ISO_3166-1_alpha-2#Officially_assigned_code_elements).
 currency                        | String   | Read       | Rental's currency code, list of codes described in [enums section](/reference/enums#currencies).
+damage_deposit                  | [Decimal](/reference/enums#formats)    | Read       | Rental's required damage deposit.
```

Comment:
Maybe it would be better to drop `required`, it might confuse some API doc readers that this attribute is required as in fact it's not mandatory (same in rentals_read and rentals_write scopes part).

Reply:
It was already there like this: https://github.com/BookingSync/bookingsync-api-docs/commit/15052cb4930f1ab6c5e32bb5361c0107d26dd13a#diff-087e8519e3d1e2e0abaa6bf72d056066da650753d416eb0828274e7016e8ff78R242

Reply:
Hmmm not a big fan of it, but it kinda makes sense as well, so I won't insist on the change.